src/frontend/src/players/player.ts

```diff
@@ -99,4 +99,21 @@ export class Player extends AbstractPlayer {
       this.playPipelineTail.setSubtitlesFromPid(subtitlesPid)
     }
   }
+
+  /* Should bo instantiate in FLOW-5509 Bedside FlowR v5 | Barco | Play/Pause*/
+  /* eslint-disable @typescript-eslint/no-unused-vars */
+  backToLive(event: Electron.IpcMainEvent): void | Promise<void> {
+    throw "not available"
+    return undefined;
```

Comment:
Why is there a return after throwing ?
Additionally, use an `Error` object when `throw`ing. That's the object any caller using `catch` will expect.